### ISSUE TITLE
academic request

### ISSUE THREAD

**Issue by Weimap** [2154022466/Meta-Learning4FSTSF#2]
Hi！Thanks for the code, I have a question for you。
In the path 'few_shot_data\\rain_data.pkl' under the embedding.py file, there will be an error that there is no such path. Is it necessary to specify a specific file? For example 'few_shot_data/test_data_embedding_10.pkl' Is it?

**Comment by xiaofeng-github** [comment]
To be honest, I do not understand your question clearly, well, some suggestions can provide for you. If you use some new dataset instead of from the repositoty, you need data_preprocess.py and embedding .py to handle your dataset and get the training set and test set or you can wirte own preprocess module; If you only want to try running the code, you do not need focus the preprocess module provided in this repository and concrete information can be find in the ReadMe.

**Comment by Weimap** [comment]
Hello, thank you for your reply. I only want to use the UCR dataset at the moment, but I found that when data_preprocessing.py performs data processing, I don't understand what the function construct_dataset is doing. There is also a question, few_shot_load_data.pkl in the get_basic_data function what data to load. In my understanding, if you just use the UCR data, you only need to process it through the few_shot_data function, is that so? Can you help me with the questions I have? thank you very much!


`def construct_dataset(size=100):`

` load_data_path = osp.join(DATADIR, 'few_shot_data/few_shot_load_data.pkl')`

**Comment by K0824-bryant** [comment]
> 您好，感谢您的回复。我目前只想使用 UCR 数据集，但我发现data_preprocessing.py执行数据处理时，我不明白construct_dataset函数在做什么。还有一个问题，few_shot_load_data.pkl在get_basic_data函数中加载什么数据。在我的理解中，如果你只是使用UCR数据，你只需要通过few_shot_data函数来处理它，是这样吗？你能帮我解决我的问题吗？谢谢！
> 
> `def construct_dataset(size=100):`
> 
> ` load_data_path = osp.join(DATADIR, 'few_shot_data/few_shot_load_data.pkl')`

**Comment by K0824-bryant** [comment]
您好 请问您弄明白了吗？ 我和你有一样的问题，能否交流一下？
